### PR TITLE
Add commonly used properties for genomic entities

### PR DESCRIPTION
**Pull request by deepakunni3**

### CODE REVIEW

File: biolink-model.yaml [hunk 1]
```diff
@@ -1941,11 +1950,28 @@ slots:
...
     mappings:
+      - skos:altLabel
+      - gff3:Alias
+      - alliancegenome:synonyms
       - oboInOwl:hasExactSynonym
       - oboInOwl:hasNarrowSynonym
       - oboInOwl:hasBroadSynonym
       - oboInOwl:hasRelatedSynonym
 
+  xref:
```
Comment (cmungall) [inline biolink-model.yaml:1961]: I assumed we had this already!

File: biolink-model.yaml [hunk 2]
```diff
@@ -3317,6 +3343,14 @@ classes:
...
     is_a: gene or gene product
     aliases: ['locus']
+    slots:
+      - id
+      - name
+      - synonym
+      - xrefs
+    slot_usage:
+      - name:
+          description: Gene symbol
```
Comment (cmungall) [inline biolink-model.yaml:3353]: this is terminologically awkward as the full name (e.g. "sonic hedgehog") is sometimes referred to as 'name'

Comment (cmungall) [reply biolink-model.yaml:3353]: we'll also want description

File: biolink-model.yaml (hunk 1 above)
Comment (cmungall) [reply biolink-model.yaml:1961]: the mappings model from the blml metamodel is an importable yaml..

this is also the opportunity to do this right. We can have an sssom like model where the mapping is an object, with a type, provenance, comments, etc...

**Comment by TomConlin** [comment]
dbxref  is common

**Pull request closed without merging**